main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -98,6 +98,8 @@
 
     public static final int DAYS_TO_SECONDS = 24 * 60 * 60;
 
+    private static final int HISTORY_SIZE = 10;
```

Comment:
still 10?

Reply:
Oh 😅 I guess it makes sense to commit all changes before pushing :)

This happens when coding on two topics at the same time... 😂

Reply:
;-)